src/draw/control/DrawToolbar.Control.js

```diff
@@ -0,0 +1,33 @@
+L.DrawToolbar.Control = L.Toolbar.Control.extend({
+	options: {
+		actions: [
+			L.Draw.Polygon,
```

Comment:
How can a user of leaflet.draw pass in options to the draw handlers? E.g. the `shapeOptions`